game/world/managers/objects/units/UnitManager.py

```diff
@@ -154,6 +154,7 @@ def __init__(self,
         self.object_type_mask |= ObjectTypeFlags.TYPE_UNIT
         self.update_packet_factory.init_values(UnitFields.UNIT_END)
 
+        self.time_since_damage_taken = 0
```

Comment:
From 0.5.4 patch notes:
- Creature pursuit is now timer based rather than distance based.